errbot/errBot.py

```diff
@@ -524,6 +524,15 @@ def callback_message(self, mess):
                 except Exception:
                     log.exception("Crash in a callback_message handler")
 
+    def callback_mention(self, mess, people):
```

Comment:
maybe reuse _dispatch_to_plugins ? (we should change also the callback_message but this is for another PR)